src/Licenses/Entities/License/Prediction.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace ArrowSphere\PublicApiClient\Licenses\Entities\License;
+
+use ArrowSphere\PublicApiClient\AbstractEntity;
+use ArrowSphere\PublicApiClient\Exception\EntityValidationException;
+
+/**
+ * Class ActiveSeats
+ */
+class Prediction extends AbstractEntity
+{
+    public const COLUMN_CURRENCY = 'currency';
+    public const COLUMN_UPDATE_DATE_OF_ML_SCRIPT = 'updatedAt';
+    public const COLUMN_LICENSE_REFERENCE = 'licenseRef';
+    public const COLUMN_PREDICTION_DATA = 'prediction';
+    protected const VALIDATION_RULES = [
+        self::COLUMN_CURRENCY => 'present',
+        self::COLUMN_UPDATE_DATE_OF_ML_SCRIPT => 'present',
+        self::COLUMN_LICENSE_REFERENCE => 'present',
+        self::COLUMN_PREDICTION_DATA =>'present'
+    ];
+
+    /**
+     * @var string
+     */
+    private $currency;
+
+    /**
+     * @var string
+     */
+    private $updatedAt;
+
+    /**
+     * @var string
+     */
+    private $licenseRef;
+
+    /**
+     * @var PredictionData[]
+     */
+    private $predictionData;
+
+    /**
+     * Predictions constructor
+     *
+     * @param array $data
+     *
+     * @throws  EntityValidationException
+     */
+    public function __construct(array $data)
+    {
+        parent::__construct($data);
+
+        $this->currency = $data[self::COLUMN_CURRENCY];
+        $this->updatedAt = $data[self::COLUMN_UPDATE_DATE_OF_ML_SCRIPT];
+        $this->licenseRef = $data[self::COLUMN_LICENSE_REFERENCE];
+        $this->predictionData = $data[self::COLUMN_PREDICTION_DATA];
+    }
+
+    /**
+     * Predictions constructor
+     *
+     * @return  string
+     */
+    public function getCurrency(): string
+    {
+        return $this->currency;
+    }
+
+    /**
+     * @return string
+     */
+    public function getUpdateDate(): string
+    {
+        return $this->updatedAt;
+    }
+
+    /**
+     * @return string
+     */
+    public function getLicenseRef(): string
+    {
+        return $this->licenseRef;
+    }
+
+    /**
+     * @return array
```

Comment:
```suggestion
     * @return PredictionData[]
```
or Prediction[] if you follow my comment on top of this file